src/estimators/tests/any_sketch_test.py

```diff
@@ -37,6 +38,13 @@ def test_uniform_distribution(self):
     self.assertEqual(u1, u2)
     self.assertEqual(u1.get_index(42), 2)
 
+  def test_geometric_distribution(self):
```

Comment:
Can you add a parameterized test as below so that a few different values can be tested?